gui/encyclopedia/articles/ancient_world/warfare/history_of_arrowheads.json

```diff
@@ -0,0 +1,5 @@
+{
+    "title": "Production of arrowheads in antiquity",
+    "subtitle": "A story from stone collecting to metal casting",
+    "content": "Humans' first arrowheads were just small stones, bones, or really any sharp and pointed object found in the wild. Eventually, as humans learned how to melt and form metals and later how to consistently extract them from ore bodies, they also began shaping arrowheads from this new material. Earliest findings date back to the 3rd and 4th millenia BC and were made of copper. Such, however, were, at latest by the year 2200 BC, entirely replaced by the new and much harder metal of bronze. From 1300 BC onwards, arrowheads of iron emerged and started being used alongside the bronze ones. While iron was, due to its higher melting point, harder to form than bronze, it also proved much stronger and lighter. And arrowheads are comparably simple shapes anyway. This spread was also driven by the Bronze Age collapse, after which the acquisition of tin (necessary for bronze) was impeded by a decline in trade activity. However, bronze arrowheads coexisted for a very long time. Because of how easy casting bronze was, they could be mass produced, and soldiers were able to create their arrows fully by themselves. And bronze was also the material for special arrows that required complex-shaped heads."
```

Comment:
```suggestion
    "content": "Humans' first arrowheads were just small stones, bones, or any sharp and pointed object found in the wild. Eventually, as humans learned how to melt and form metals and later how to consistently extract them from orebodies, they also began shaping arrowheads from this new material. Earliest findings date back to the 3rd and 4th millennia BC and were made of copper. Such, however, were, at latest by the year 2200 BC, entirely replaced by the new and much harder metal of bronze. From 1300 BC onwards, arrowheads of iron emerged and started being used alongside the bronze ones. While iron was, due to its higher melting point, harder to form than bronze, it also proved much stronger and lighter. And arrowheads are comparably simple shapes anyway. This spread was also driven by the Bronze Age collapse, after which the acquisition of tin (necessary for bronze) was impeded by a decline in trade activity. However, bronze arrowheads coexisted for a very long time. Because of how easy casting bronze was, they could be mass-produced, and soldiers were able to create their arrows fully by themselves. And bronze was also the material for special arrows that required complex-shaped heads."
```